src/hivpy/population.py

```diff
@@ -183,6 +183,12 @@ def get_sub_pop_intersection(self, subpop_1, subpop_2):
         """
         return pd.Index.intersection(subpop_1, subpop_2)
 
+    def get_sub_pop_union(self, subpop_1, subpop_2):
```

Comment:
Make this variadic (so we're not calling unions in unions)? Or potentially just merge `risk_informed` and `suspect_risk` sub-pops into one sub-pop.